crates/core/src/repository/mod.rs

```diff
@@ -13,15 +13,16 @@ use std::str::FromStr;
 mod backend;
 use backend::Backend;
 
-pub use backend::{curse, git, townlongyak, tukui, wowi};
-use backend::{Curse, Github, Gitlab, TownlongYak, Tukui, WowI};
+pub use backend::{curse, git, hub, tukui, wowi};
+use backend::{Curse, Github, Gitlab, Hub, Tukui, WowI};
 
 #[derive(Debug, Clone, Copy, PartialEq, PartialOrd, Ord, Eq, Serialize, Deserialize)]
 pub enum RepositoryKind {
     Curse,
     Tukui,
     WowI,
-    TownlongYak,
+    #[serde(alias = "TownlongYak")]
+    Hub,
```

Comment:
We can do this to make sure that addons installed from TownlongYak will continue and use Hub going forward. This is because we used TLY with Hub as a backend before, so they are plug and play.

If we remove it, the addons will fall back to Curse. This will actually still work because TLY addons recently were added to Curse.

@tarkah Which migration solution do you like the best? Rely on fallback or add the alias.

Reply:
I like the alias idea, since that's the source those add-ons are hooked up to anyway 👍